app/controllers/devise_overrides/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,36 @@
+class DeviseOverrides::OmniauthCallbacksController < DeviseTokenAuth::OmniauthCallbacksController
+  def omniauth_success
+    get_resource_from_auth_hash
+
+    if @resource.nil?
+      redirect_to "#{ENV.fetch('FRONTEND_URL', nil)}/app/login?error=oauth-no-user"
+      return
+    end
```

Comment:
@sojan-official @pranavrajs need your thoughts on this approach, to reject a login

Reply:
Can we show a toast when there is an error?

Reply:
We raise a toast that says "User not found. You need to sign up first"

<img width="1048" alt="Screenshot 2023-01-29 at 11 18 12 AM" src="https://user-images.githubusercontent.com/18097732/215307677-8a44bfaf-f7b7-4e16-986b-f4f2931778c9.png">